main.tf

```diff
@@ -1,9 +1,9 @@
 terraform {
-  required_version = "~>0.11"
+  required_version = "0.12.18"
```

Comment:
@DazKins To be honest I've haven't used TF for a long time, setting required version down to minor version might be a bit strong? could this be something like `~0.12` ?

Reply:
Sounds good. Changed now